src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -209,6 +261,20 @@ limitations under the License.
             help="Remove MTU">
             <ACTION>python $SONIC_CLI_ROOT/sonic-cli-if.py patch_openconfig_interfaces_interfaces_interface_config_mtu ${iface} 9100 </ACTION>
         </COMMAND>
+        <COMMAND
+            name="channel-group"
+            help="Configure port channel parameters">
+	    <PARAM 
+                name="lag-id" 
+                help="Specify a port channel number"
+                ptype="UINT" />
```

Comment:
"channel-group <lag-id> mode active" should be followed to match the IS-CLI, this will clearly indicate that the lag we're configuring is LACP port-channel.

Reply:
instead of UINT, define the range for port-channel in Sonic_types.h and use it here.

Reply:
> instead of UINT, define the range for port-channel in Sonic_types.h and use it here.

    Added in sonic_types.xml : 
       <PTYPE
+        name="LAG_ID"
+        method="integer"
+        pattern="1..128"
+        help=""
+        />

Reply:
> "channel-group mode active" should be followed to match the IS-CLI, this will clearly indicate that the lag we're configuring is LACP port-channel.
Added: 
![image](https://user-images.githubusercontent.com/14809064/65734973-40df5e80-e08a-11e9-9443-f579d948752e.png)

Reply:
> instead of UINT, define the range for port-channel in Sonic_types.h and use it here.

Added:
    <PTYPE
        name="LAG_ID"
        method="integer"
        pattern="1..128"
        help=""
        />
    <!--=======================================================-->
    <PTYPE
        name="MIN_LINK"
        method="integer"
        pattern="1..32"
        help=""
        />